### PR TITLE
SolarNuAna

### PR DESCRIPTION
**Pull request by mantheys**
Update latest chnages to SolarNuAna workflow and fix cmakelists dependecies.

### CODE REVIEW

File: duneana/WireAna/WireAnaTree_module.cc [hunk 1]
```diff
@@ -87,16 +87,23 @@ void wire::WireAnaTree::endJob(){
 }
 void wire::WireAnaTree::analyze(art::Event const& e)
 {
+  // Print a hello message.
```
Comment (tomjunk) [inline duneana/WireAna/WireAnaTree_module.cc]: Might I suggest putting these cout statements in a switchable debug-only mode?

File: duneana/SolarNuAna/AdjHitsUtils.cc [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+          AdjHitVec.push_back(MyVec[AddNow[AddNow.size() - 1 - aa]]);
+          MyVec.erase(MyVec.begin() + AddNow[AddNow.size() - 1 - aa]); // This line creates segmentation fault
+                                                                      // std::cout << "Erase works" << std::endl;
+        }
+
+        LastSize = NewSize;
+        NewSize = AdjHitVec.size();
+        if (HeavDebug)
+        {
+          std::cerr << "\t---After that pass, AddNow was size " << AddNow.size() << " ==> LastSize is " << LastSize << ", and NewSize is " << NewSize
```
Comment (tomjunk) [inline duneana/SolarNuAna/AdjHitsUtils.cc]: Why the mixture of cout and cerr?  You have to look in two places to find your output, and you may lose the ordering of the messages, even if they are combined together e.g. with bash's 2>&1.  I don't know; I haven't tried, but due to output file buffering, I'd imagine you'd get an arbitrary mixture.

File: duneana/SolarNuAna/AdjHitsUtils.cc [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+        double channel = 0;
+        double tick = 0;
+
+        for (recob::Hit TPCHit : hits)
+        {
+          tick += TPCHit.Integral() * TPCHit.PeakTime();
+          channel += TPCHit.Integral() * TPCHit.Channel();
+          adcInt += TPCHit.Integral();
+        }
+        tick /= adcInt;
```
Comment (tomjunk) [inline duneana/SolarNuAna/AdjHitsUtils.cc]: check for divide by zero?

**Comment by mantheys** [comment]
Hi @tomjunk thanks for your comments. I've added a new commit "FIX PR COMMENTS" that should take care of them. Let me know if you see anything else. Thanks!!!

**Review by LauPM — APPROVED** [review]

**Pull request merged**